bin/check.js

```diff
@@ -14,8 +14,11 @@ program
   .option('--no-skipped', 'throw error if skipped tests found')
   .option('--typescript', 'enable typescript support')
   .action((framework, files, opts) => {
-    
-    const analyzer = new Analyzer(framework, opts.dir || process.cwd());
+    let dir = undefined
+    if (opts.dir) {
+      dir = opts.dir.endsWith('/') ? opts.dir : `${opts.dir}/`;
```

Comment:
this seems like a dirty fix
you should not use `/` here as it is platform-dependent. I think it should be fixed inside Analyzer, using `path.join`

Reply:
@DavertMik Checked the analyzer and our code is working because of this line

```js
file.replace(this.workDir + path.sep, '');
```

Case 1: consider workDir  as `folder`
Now it adds path.sep and replaces it with empty string ie : `folder/` => ''

Case 2: Consider workDir as `folder/`
Now it adds extra / so the name is not replaced `folder//`

So this is happy coincidence that it works. So instead of using `/` I can use path.sep to fix this. Thought?